cartography/intel/gcp/bigquery.py

```diff
@@ -81,7 +81,7 @@ def transform_bigquey_dataset(bigquery: Resource, datasets: List[Dict], project_
         dataset['id'] = dataset.get('datasetReference', {}).get('datasetId', '')
         dataset['uniqueId'] = f"projects/{project_id}/datasets/{dataset['id']}"
         dataset['details'] = get_dataset_info(bigquery, dataset['id'])
-        # Console Link to be implemented
+        dataset['consolelink'] = gcp_console_link.get_console_link(project_id=project_id, resource_name='bigquery')
```

Comment:
we don't have support for bigquery in cloudconsolelink project. we should add support for bigquery in cloudconsolelink project first. Otherwise, this will throw an exception in production.

Reply:
I have added the the consolelink implementeation for new resources and raised a PR